core/src/main/java/org/elasticsearch/rest/BytesRestResponse.java

```diff
@@ -127,16 +138,13 @@ private static XContentBuilder convert(RestChannel channel, RestStatus status, T
             builder.startArray();
             for (ElasticsearchException rootCause : rootCauses){
                 builder.startObject();
-                rootCause.toXContent(builder, new ToXContent.DelegatingMapParams(Collections.singletonMap(ElasticsearchException.REST_EXCEPTION_SKIP_CAUSE, "true"), channel.request()));
+                rootCause.toXContent(builder, new ToXContent.DelegatingMapParams(Collections.singletonMap(ElasticsearchException.REST_EXCEPTION_SKIP_CAUSE, "true"), params));
```

Comment:
this is only for the root cause - we want this to be small and crisp. The full errors below will use the default